zbus/src/address/transport/unixexec.rs

```diff
@@ -0,0 +1,114 @@
+use std::{
+    borrow::BorrowMut, ffi::OsString, fmt::Display, os::unix::ffi::OsStrExt, path::PathBuf,
+    process::Stdio,
+};
+
+use crate::process::Command;
+
+use super::encode_percents;
+
+/// `unixexec:` D-Bus transport.
+///
+/// <https://dbus.freedesktop.org/doc/dbus-specification.html#transports-exec>
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct Unixexec {
+    path: PathBuf,
+    arg0: Option<OsString>,
+    args: Vec<String>,
```

Comment:
The both should use the same String type, no?

Reply:
Good catch! Fixed in latest revision.

Reply:
Thanks but seems you went for the opposite of what I'd have expected. :) Wouldn't `OsString` not be better here? :thinking:

Reply:
No worries at all. Pushed a new revision where all the `Unixexec` args are now `OsString`s.